thirdeye/thirdeye-pinot/src/test/java/com/linkedin/thirdeye/tools/CleanupAndRegenerateAnomaliesTool.java

```diff
@@ -168,6 +204,10 @@ private void deleteRawResults(List<RawAnomalyResultDTO> rawResults) {
   private void deleteMergedResults(List<MergedAnomalyResultDTO> mergedResults) {
     LOG.info("Deleting merged results");
     for (MergedAnomalyResultDTO mergedResult : mergedResults) {
+      // Delete raw anomalies of the merged anomaly
```

Comment:
That's a good use case. We can add that flag when we want to conduct experiment on merge logic. Don't add it now because I don't want that flag to be misused.